src/main.ts

```diff
@@ -0,0 +1,10 @@
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app.module";
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+async function bootstrap() {
```

Comment:
```suggestion
async function bootstrap() {
```
see what app.listen() is returning